content/fr/docs/setup/pick-right-solution.md

```diff
@@ -0,0 +1,302 @@
+* [Containership](https://containership.io/containership-platform)
+* [Docker Enterprise](https://www.docker.com/products/docker-enterprise)
+* [Gardener](https://gardener.cloud/)
+* [Giant Swarm](https://giantswarm.io)
+* [Google Compute Engine (GCE)](/docs/setup/turnkey/gce/)
+* [IBM Cloud](https://github.com/patrocinio/kubernetes-softlayer)
+* [Kontena Pharos](https://kontena.io/pharos/)
+* [Kubermatic](https://cloud.kubermatic.io)
+* [Kublr](https://kublr.com/)
+* [Madcore.Ai](https://madcore.ai/)
+* [Nirmata](https://nirmata.com/)
+* [Nutanix Karbon](https://www.nutanix.com/products/karbon/)
+* [Oracle Container Engine for K8s](https://docs.us-phoenix-1.oraclecloud.com/Content/ContEng/Concepts/contengprerequisites.htm)
+* [Pivotal Container Service](https://pivotal.io/platform/pivotal-container-service)
+* [Rancher 2.0](https://rancher.com/docs/rancher/v2.x/en/)
+* [Stackpoint.io](/docs/setup/turnkey/stackpoint/)
+* [Tectonic by CoreOS](https://coreos.com/tectonic)
+* [VMware Cloud PKS](https://cloud.vmware.com/vmware-cloud-pks)
+
+## Solutions On-Premises clés en main
+
+Ces solutions vous permettent de créer des clusters Kubernetes sur votre cloud privé sécurisé avec seulement quelques commandes.
+
+* [Agile Stacks](https://www.agilestacks.com/products/kubernetes)
+* [APPUiO](https://appuio.ch)
+* [Docker Enterprise](https://www.docker.com/products/docker-enterprise)
+* [Giant Swarm](https://giantswarm.io)
+* [GKE On-Prem | Google Cloud](https://cloud.google.com/gke-on-prem/)
+* [IBM Cloud Private](https://www.ibm.com/cloud-computing/products/ibm-cloud-private/)
+* [Kontena Pharos](https://kontena.io/pharos/)
+* [Kubermatic](https://www.loodse.com)
+* [Kublr](https://kublr.com/)
+* [Mirantis Cloud Platform](https://www.mirantis.com/software/kubernetes/)
+* [Nirmata](https://nirmata.com/)
+* [OpenShift Container Platform](https://www.openshift.com/products/container-platform/) (OCP) by [Red Hat](https://www.redhat.com)
+* [Pivotal Container Service](https://pivotal.io/platform/pivotal-container-service)
+* [Rancher 2.0](https://rancher.com/docs/rancher/v2.x/en/)
+* [SUSE CaaS Platform](https://www.suse.com/products/caas-platform)
+* [SUSE Cloud Application Platform](https://www.suse.com/products/cloud-application-platform/)
+
+## Solutions personnalisées
+
+Kubernetes peut fonctionner sur une large gamme de fournisseurs de Cloud et d'environnements bare-metal, ainsi qu'avec de nombreux 
+systèmes d'exploitation.
+
+Si vous pouvez trouver un guide ci-dessous qui correspond à vos besoins, utilisez-le. C'est peut-être un peu dépassé, mais...
+ce sera plus facile que de partir de zéro. Si vous voulez repartir de zéro, soit parce que vous avez des exigences particulières, 
+ou simplement parce que vous voulez comprendre ce qu'il y a à l'interieur de Kubernetes
+essayez le guide [Getting Started from Scratch] (/docs/setup/scratch/).
```

Comment:
Lien qui a l'air cassé

Reply:
idem false broken link, i fix it though :)